tests/model_serving/model_server/serverless/test_odh_model_controller_failing_while_deleted_isvc.py

```diff
@@ -0,0 +1,79 @@
+import pytest
+from ocp_utilities.infra import get_pods_by_name_prefix
+from pytest_testconfig import py_config
+from simple_logger.logger import get_logger
+from timeout_sampler import TimeoutSampler, TimeoutExpiredError
+
+from tests.model_serving.model_server.utils import verify_inference_response
+from utilities.constants import (
+    ModelFormat,
+    ModelInferenceRuntime,
+    ModelVersion,
+    Protocols,
+    Timeout,
+)
+from utilities.exceptions import PodLogMissMatchError
+from utilities.inference_utils import Inference
+from utilities.manifests.onnx import ONNX_INFERENCE_CONFIG
+
+pytestmark = [
+    pytest.mark.serverless,
+    pytest.mark.sanity,
+    pytest.mark.usefixtures("valid_aws_config"),
+]
+
+LOGGER = get_logger(name=__name__)
+
+
+@pytest.mark.serverless
+@pytest.mark.parametrize(
+    "model_namespace, openvino_kserve_serving_runtime, ovms_serverless_inference_service",
+    [
+        pytest.param(
+            {"name": "serverless-maistra"},
+            {
+                "runtime-name": ModelInferenceRuntime.ONNX_RUNTIME,
+                "model-format": {ModelFormat.ONNX: ModelVersion.OPSET13},
+            },
+            {
+                "name": ModelFormat.ONNX,
+                "model-version": ModelVersion.OPSET13,
+                "model-dir": "test-dir",
+            },
+        )
+    ],
+    indirect=True,
+)
+class TestLogForOdHModelController:
+    def test_odh_model_controller_failing_while_deleted_isvc(self, ovms_serverless_inference_service):
```

Comment:
please rename; the test is not about controller failing

Reply:
yess i did it let me know  if new one is fine